doc/assets.md

```diff
@@ -1,10 +1,10 @@
-## Syntax Highlighting in bat
+## Add New Syntax Highlighting into the bat binary
 
-`bat` uses the [syntect](https://github.com/trishume/syntect) library to highlight source
+`bat` uses the excellent [syntect](https://github.com/trishume/syntect) library to highlight source
 code. As a basis, syntect uses [Sublime Text](https://www.sublimetext.com/) syntax definitions
 in the `.sublime-syntax` format.
 
-In order to add new syntaxes to `bat`, follow these steps:
+Follow these steps to add new syntaxes into the `bat` binary:
 
 1. Find a Sublime Text syntax for the given language, preferably in a separate Git repository
    which can be included as a submodule (under `assets/syntaxes`).
```

Comment:
It would be great if we could amend this with:

"Make sure that the syntax meets the criteria for inclusion within `bat` (see below)."

And then add a new section below with:

```
## Criteria for inclusion of new syntaxes

- More than 10,000 downloads on packagecontrol.io/
```